packages/vow/src/watch-utils.js

```diff
@@ -119,27 +173,36 @@ export const prepareWatchUtils = (
           }
           return kit.vow;
```

Comment:
Is there a path for typing this, so we can keep the existing `Vow<any[]>` for `allVows` and something like `Vow<({status: 'fulfilled', value: any} | {status: 'rejected', reason: Error})[]>` for `allVowsSettled`?

h/t @Chris-Hibbert for the suggestion: https://github.com/Agoric/agoric-sdk/pull/9902/files#r1757395986

Reply:
I've given this a shot above, but it is wholly untested.  Hope it helps.

Reply:
Thanks, your suggestion worked! In hindsight, it was obvious 😅